CI/values.yaml

```diff
@@ -6,17 +6,20 @@ name: grid-tariff-api
 namespace: kunde
 microserviceType: webapi
 
-replicaCount: 1
+replicaCount:
+  dev: 1
+  test: 1
+  prod: 2
 
 image:
   repository: containerregistryelvia.azurecr.io/kunde-grid-tariff-api
 
 resources:
   requests:
-    cpu: "500m"
-    memory: "500Mi"
-  limits:
     cpu: "1"
+    memory: "1Gi"
+  limits:
+    cpu: "2"
     memory: "2Gi"
```

Comment:
Hvis vi skal sikre oss mot at den går tom for minne igjen, så bør limit også økes. F.eks. til 3Gi.

Reply:
Økt til 3Gi.
Det er korrekt at flere pods ikke er optimalt realatert til gjeldende caching-strategi, men med dagens last er kosenkvensene av dette begrenset. Tilbakemelding fra Øystein@core er også at API'er bør ha to pods.